wagtail/admin/templates/wagtailadmin/pages/privacy_switch_panel.html

```diff
@@ -20,7 +20,7 @@
             </p>
 
             {% if page.id and page_perms.can_set_view_restrictions %}
-                <button type="button" data-url="{% url 'wagtailadmin_pages:set_privacy' page.id %}" data-action-set-privacy class="button">
+                <button type="button" data-a11y-dialog-show="set-privacy" class="button">
```

Comment:
Shouldn’t this template and its `PrivacyModalPanel` class be removed altogether?

Reply:
I intentionally updated the template and left `PrivacyModalPanel` in as it seems there are people who use it directly (for whatever reason): https://github.com/search?q=privacymodalpanel&type=code

P.s. I just added `min-height` to `w-dialog__content`.

Reply:
Discussed in today’s sprint planning, we’ll remove this.